File: xcel6695-cs50-2017-x-pset6/xcel6695-cs50-2017-x-pset6/greedy.py
while True:
  print("O hai! How much change is owed?")
  try:
    change = float(input())
    if change >= 0.0:
      break
  except ValueError:
    print("", end = "")

number_of_coins = 0
cents = int(round(change * 100))
number_of_coins += cents // 25
cents %= 25
number_of_coins += cents // 10
cents %= 10
number_of_coins += cents // 5
cents %= 5
number_of_coins += cents
print("{}".format(number_of_coins))

File: xcel6695-cs50-2017-x-pset1/xcel6695-cs50-2017-x-greedy/xcel6695-cs50-2017-x-greedy/greedy.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void) 
{
    float change;
    printf("O hai! How much change is owed?\n");
    change = get_float();
    while (change < 0) 
    {
        printf("How much change is owed?\n");
        change = get_float();
    }
    
    change = change * 100;
    int cents = (int) round (change);
    
    int count = 0;
    if (cents >= 25) 
    {
        count = count + cents / 25;
        cents = cents% 25;
    }
    if (cents >= 10)
    {
        count = count + cents / 10;
        cents = cents % 10;
    }
    if (cents >= 5) 
    {
        count = count + cents / 5;
        cents = cents % 5;
    }
    if (cents >= 1) 
    {
        count = count + cents;
    }
    printf("%d\n", count);
}